### ISSUE TITLE
Withdraw signature has no nonce

### ISSUE THREAD

**Issue by Marik-D** [thehubbleproject/hubble-contracts#665]
To claim the withdrawn funds to an ethereum address users submit a signature signed with their BLS wallet. Right now that signature only includes the address that the funds are being withdrawn to, and no nonce. 

This allows an attacker to re-use the same signature for the next withdrawal from the same account.

**Comment by jacque006** [comment]
This is similar to the replay attack illustrated in this Solidity example: https://docs.soliditylang.org/en/v0.6.12/solidity-by-example.html?highlight=nonce#what-to-sign

@Marik-D when referring to nonce, I assume you are referring to a data structure that would track nonce's specific to the `WithdrawlManager` (such as `mapping(uint256 => bool) usedNonces` from the example above), and not the nonce associated with L1 txns for an ETH address?

**Comment by Marik-D** [comment]
Yes, it doesn't have to be the nonce of the L1 ETH account, just some single-use value. Can be something specific to this withdraw operation.
